packages/app/src/app/pages/Sandbox/Editor/Workspace/Files/index.js

```diff
@@ -43,6 +43,7 @@ class Files extends React.Component {
         root
         getModulePath={this.getModulePath}
         title={sandbox.title || 'Project'}
+        signals={this.props.signals}
```

Comment:
Don't pass `signals` to `DirectoryEntry`, wrap it with an `inject` instead

Reply:
We have to pass it due to the temporary wrapper. With Overmind the **inject** actually does nothing. It is the observer that provides this stuff, though that can not be around the Drag HOC. Drag is a messy thing in React, this should be refactored with hooks later.

Reply:
Hmmm too bad it needs to be like this.
Maybe add an `TODO` next to it, so we don't forget about it? 🤔

Reply:
Jup :)